lib/screens/listOfOrgs/widgets.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
```

Comment:
widgets should be a folder containing files of stateless/stateful widgets
not a single file containing all widgets